docker/buildroot.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -e
+
+docker_image="goranche/raspbian-ua-netinst"
+
+if [ "${1}" != "IN_DOCKER" ]; then
+	docker run --privileged -ti --rm -v $(pwd):/raspbian-ua-netinst "${docker_image}"
+	exit
+fi
+
```

Comment:
well it isn't shown here but line 7 says `if [ "${1}" != "IN_DOCKER" ]; then`

So I was guessing there would be a CLI parameter.